modules/sms_email_reminder/cron_email_notification.php

```diff
@@ -7,11 +7,14 @@
 //
 // Created by:
 // Updated by:	Larry Lart on 10/03/2008
+// Updated by:  Daniel Pflieger on 7/16/2017 daniel@growlingflea.com growlingflea@gmail.com daniel@mi-squared.com
 ////////////////////////////////////////////////////////////////////
 
 // larry :: hack add for command line version
 $_SERVER['REQUEST_URI']=$_SERVER['PHP_SELF'];
 $_SERVER['SERVER_NAME']='localhost';
+$_SERVER['HTTP_HOST']='default';
+$ignoreAuth = true;
```

Comment:
OK.  The idea is that this should be run from the command line.  I guess I'm not sure which exit trick you are referring to, but I'll take a look at the other script.